Source/WebCore/platform/graphics/displaylists/DisplayListItems.cpp

```diff
@@ -860,6 +874,12 @@ void dumpItem(TextStream& ts, const ClearRect& item, OptionSet<AsTextFlag>)
     ts.dumpProperty("rect", item.rect());
 }
 
+void dumpItem(TextStream& ts, const DrawControlPart& item, OptionSet<AsTextFlag>)
+{
+    ts.dumpProperty("type", item.type());
+    ts.dumpProperty("rect", item.rect());
```

Comment:
You will probably help your future self by dumping the `ControlStyle` and m_deviceScaleFactor

Reply:
Fixed.